modules/external-api/src/main/java/org/opencastproject/external/endpoint/ListProviderEndpoint.java

```diff
@@ -94,26 +93,28 @@ void activate(ComponentContext cc) {
 
   @GET
   @Path("providers.json")
-  @Produces(MediaType.APPLICATION_JSON)
+  @Produces({ ApiMediaType.JSON, ApiMediaType.VERSION_1_10_0 })
   @RestQuery(name = "availableProviders", description = "Provides the list of the available list providers", responses = { @RestResponse(description = "Returns the availables list providers.", responseCode = HttpServletResponse.SC_OK) }, returnDescription = "")
-  public Response getAvailablesProviders(@Context HttpHeaders headers) {
+  public Response getAvailablesProviders(@HeaderParam("Accept") String acceptHeader) {
+    final ApiVersion requestedVersion = ApiMediaType.parse(acceptHeader).getVersion();
```

Comment:
```suggestion
```

This is a nitpick, maybe this is fair, because we want to do something with the parsed version in the future, but right now we don't, so this is unnecessary, because ...